src/controllers/connection/newConnection.ts

```diff
@@ -363,7 +387,6 @@ export class NewConnectionController extends HTMLController {
   }
 
   private async sendNewConnectionEmail(email: string, invite: { companyNumber: string; inviteUrl: string }) {
-    this.logger.debug('NEW_CONNECTION: sending emails')
```

Comment:
is this not a useful log? Or is the information  that we are attempting to send a newConnectionEmail being logged somewhere else already?

Reply:
it's being logged at the earlier state, where we call that function

https://github.com/digicatapult/veritable-ui/blob/a08f82a03f494886015787884761c62069521013/src/controllers/connection/newConnection.ts#L202-L205